openwisp_users/admin.py

```diff
@@ -142,6 +142,15 @@ class UserAdmin(MultitenantAdminMixin, BaseUserAdmin, BaseAdmin):
                     'last_login']
     inlines = [EmailAddressInline, OrganizationUserInline]
     save_on_top = True
+    actions = ['make_inactive', 'make_active']
+
+    def make_inactive(self, request, queryset):
+        queryset.update(is_active=False)
+    make_inactive.short_description = 'Deactivate account(s)'
```

Comment:
use `gettext_lazy` to make this translatable:

```python
make_inactive.short_description = _('Flag selected users as inactive')
```

Everytime you write anything that will be presented to the user, you must do this.

Note how I changed the string too, I want to avoid "Deactivate" because it's too similar to "Delete" which is the other admin action users may click by mistake on delete instead of deactivate.

I also removed `(s)` for consistency with `Delete selected users`.

Reply:
@nemesisdesign thanks for reviewing, though i had previously used a similar string
`make_inactive.short_description = 'make selected users inactive'` but was suggested to use this one so went for it:smile:. Also, I have implemented your suggested changes and attached pics for reference.

Reply:
ok, well the good thing is that at least is not a lot of work to change it. @atb00ker keep in mind this for future refrence, always keep consistency with what's already present in the system